R/mcmc.R

```diff
@@ -1,17 +1,52 @@
 
-#' Title - TODO
+#' Fit the base imputation model using a Bayesian approach
+#'
+#' @description
+#' `fit_mcmc()` fits the base imputation model using a Bayesian approach. This is done through a MCMC method that is implemented in `stan`
+#' and is run by using the function [rstan::sampling()].
+#' The function returns the draws from the posterior distribution of the model parameters
+#' and the `stanfit` object. Additionally it performs multiple diagnostics checks of the chain
+#' and returns warnings in case of any detected issues.
+#'
+#' @param designmat The design matrix of the fixed effects.
+#' @param outcome The response variable. Must be numeric.
+#' @param group The group variable.
```

Comment:
Maybe worth stating what types these need to be,  I.e. I think group and visit need to be factor ?